airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/setup_and_teardown_runner.py

```diff
@@ -17,16 +17,15 @@
 async def _build_container(dagger_client: dagger.Client, dockerfile_path: Path) -> dagger.Container:
     workspace = (
         dagger_client.container()
-        .with_directory("/tmp", dagger_client.host().directory(os.path.dirname(dockerfile_path)))
-        .with_workdir("/tmp")
-        .with_file("/tmp/Dockerfile.cat_setup_teardown", dagger_client.host().file(str(dockerfile_path.expanduser())))
-        .directory("/tmp")
+        .with_mounted_directory("/tmp/setup_teardown_context", dagger_client.host().directory(os.path.dirname(dockerfile_path)))
+        .directory("/tmp/setup_teardown_context")
     )
     return await dagger_client.container().build(context=workspace, dockerfile="Dockerfile.cat_setup_teardown")
 
 
-async def _build_setup_container(dagger_client: dagger.Client, dockerfile_path: Path) -> dagger.Container:
-    return await _build_container(dagger_client, dockerfile_path)
+async def _build_setup_container(dagger_client: dagger.Client, connector_path: Path, dockerfile_path: Path) -> dagger.Container:
```

Comment:
naming nit: is this this only the "setup" container, it's also use for teardown isn't it?

Reply:
True, and it also is used to send commands during test execution. Renaming to `ClientContainer` but let me know if you have a better idea.